pkgs/development/python-modules/aioasuswrt/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, asyncssh
+, buildPythonPackage
+, fetchFromGitHub
+, pytest-asyncio
+, pytest-cov
+, pytest-mock
+, pytest-runner
+, pytestCheckHook
+, pythonAtLeast
+}:
+
+buildPythonPackage rec {
+  pname = "aioasuswrt";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "kennedyshead";
+    repo = pname;
+    rev = "V${version}";
+    sha256 = "1vqihb5fgphb03867b90aayfnnrssda7vs97f84q7s4cwgjxi9lg";
+  };
+
+  nativeBuildInputs = [ pytest-runner ];
```

Comment:
I think that requirement should be patched out.

Reply:
Removed by patch now.

https://github.com/kennedyshead/aioasuswrt/pull/63